docs/getting-started.md

```diff
@@ -1,5 +1,8 @@
 # Getting Started
 
+<iframe width="1280" height="500" src="https://www.youtube.com/embed/BiytYu3EefY" title="Intro to BitOps" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
any reason not to put this embed on the front page of bitops.sh?

Reply:
Let's merge the Readme update and do the landing page in the next PR: https://github.com/bitovi/bitops/pull/389 as I'd like more there with the list of bitops values and to align that w video